FiiPrezent.Tests/EventsServiceTests.cs

```diff
@@ -1,40 +1,42 @@
 using FiiPrezent.Services;
-using Shouldly;
-using Xunit;
+using NUnit.Framework;
 
 namespace FiiPrezent.Tests
 {
+    [TestFixture]
     public class EventsServiceTests
     {
-        [Fact]
-        public void WhenTryingToRegisterWithAnInvalidCodeReturnsError()
+        private EventsService _service;
+
+        [SetUp]
+        public void SetUp()
         {
-            var service = new EventsService();
-            var result = service.RegisterParticipant("bad code", "test participant");
-            result.ShouldBeNull();
+            _service = new EventsService();
+            _service.InitializeEvents();
         }
 
-        [Fact]
-        public void WhenTryingToRegisterWithAValidCodeReturnsSuccess()
+        [Test]
+        public void RegisterParticipant_WhenCodeIsInvalid_ReturnNull()
         {
-            var service = new EventsService();
-            var result = service.RegisterParticipant("cometothedarksidewehavecookies", "test participant");
-            result.ShouldNotBeNull();
-            
+            var result = _service.RegisterParticipant("bad code", "test participant");
+
+            Assert.That(result, Is.Null);
```

Comment:
Shouldly works with any testing framework and does something that the regular asserts do not. It includes the expression on the left side of the "should" in the message (see docs for an example).
Why the change?

Reply:
I am more familiar with this one.

Reply:
Not a valid reason to change it.